pipeline/trusted-resources/cmd/sign/sign_test.go

```diff
@@ -91,94 +77,55 @@ var (
 	}
 )
 
-func TestSign_Taskrun(t *testing.T) {
+func TestSignTask(t *testing.T) {
 	ctx := context.Background()
 
 	sv, err := trustedtask.GetSignerVerifier(password)
 	if err != nil {
 		t.Fatalf("error get signerverifier: %v", err)
 	}
 
-	tr := &v1beta1.TaskRun{
-		TypeMeta:   trTypeMeta,
-		ObjectMeta: trObjectMeta,
-		Spec: v1beta1.TaskRunSpec{
-			TaskSpec: &ts.Spec,
-		},
-	}
-
 	var writer bytes.Buffer
-
-	if err := Sign(ctx, tr, sv, &writer); err != nil {
+	ts := getTask()
+	if err := SignTask(ctx, ts, sv, &writer); err != nil {
 		t.Fatalf("Sign() get err %v", err)
 	}
 
 	signed := writer.Bytes()
-	tr, signature := unmarshal(t, signed)
+	ts, signature := unmarshalTask(t, signed)
 
-	if err := trustedtask.VerifyInterface(ctx, tr.Spec.TaskSpec, sv, signature); err != nil {
+	if err := trustedtask.VerifyInterface(ctx, ts, sv, signature); err != nil {
 		t.Fatalf("VerifyTaskOCIBundle get error: %v", err)
 	}
 
 }
 
-func TestSign_OCIBundle(t *testing.T) {
-	ctx := context.Background()
-
-	sv, err := trustedtask.GetSignerVerifier(password)
-	if err != nil {
-		t.Fatalf("error get signerverifier: %v", err)
+func unmarshalTask(t *testing.T, buf []byte) (*v1beta1.Task, []byte) {
```

Comment:
Since this is doing more than just unmarshalling - add a comment explaining the behavior.